tests/agent_server/test_server_agent.py

```diff
@@ -1501,9 +1501,11 @@ async def test_autostart_mapping_update_uri(server, client, environment, async_f
     async_finalizer(a.stop)
 
     # Wait until agent is up
+    async def agent_in_db() -> None:
```

Comment:
```suggestion
    async def agent_in_db() -> bool:
```

Reply:
oh my, feels like monday morning. 

Thanks for catching this.